contracts/core/Resolving/SportsAMMV2ResultManager.sol

```diff
@@ -253,6 +257,10 @@ contract SportsAMMV2ResultManager is Initializable, ProxyOwned, ProxyPausable, P
                 if (!areResultsPerMarketSet[gameId][typeId][playerId]) {
                     resultsPerMarket[gameId][typeId][playerId] = results;
                     areResultsPerMarketSet[gameId][typeId][playerId] = true;
+                    if (numOfTicketsToExercise > 0) {
+                        address[] memory activeTickets = manager.getActiveTicketsPerMarket(0, 100, gameId, typeId, playerId);
```

Comment:
100 tickets might be too much as all tickets would have to be iterated on so it would become the main cost driver.
That said its very unlikely there would be 100 tickets for any market.